mumd/src/state.rs

```diff
@@ -45,13 +43,11 @@ pub enum ExecutionContext {
     ),
 }
 
-#[derive(Clone, Debug, Eq, PartialEq)]
+#[derive(Clone, Copy, Debug, Eq, PartialEq)]
```

Comment:
Does this type have to be `Copy` or is `Clone` sufficient?

Reply:
We copy here:
https://github.com/sornas/mum/blob/2264ce4eb7922d9c7313d64f85649e861207842b/mumd/src/state.rs#L596-L598
but we can clone if you feel strongly about it. My thinking is that it is such a trivial struct that Copy is fine.

Reply:
Seems fine.